src/server/cmd/pachd/main.go

```diff
@@ -61,15 +64,110 @@ type appEnv struct {
 	BlockCacheBytes       string `env:"BLOCK_CACHE_BYTES,default=5G"`
 	PFSCacheBytes         string `env:"PFS_CACHE_BYTES,default=1G"`
 	WorkerImage           string `env:"WORKER_IMAGE,default="`
+	WorkerSidecarImage    string `env:"WORKER_SIDECAR_IMAGE,default="`
 	WorkerImagePullPolicy string `env:"WORKER_IMAGE_PULL_POLICY,default="`
 	LogLevel              string `env:"LOG_LEVEL,default=info"`
 }
 
 func main() {
-	cmdutil.Main(do, &appEnv{})
+	switch mode {
+	case "full":
+		cmdutil.Main(doFullMode, &appEnv{})
+	case "pfs":
+		cmdutil.Main(doPFSMode, &appEnv{})
+	default:
+		fmt.Println("unrecognized mode: %s", mode)
+	}
+}
```

Comment:
That's what I did initially but it turned out to be harder than expected.  The tricky part is that there are all these inner dependencies between the servers so it's actually pretty hard to factor out each of them into their own function.

Doing it the current way is probably less bug prone anyways since it doesn't affect the `pachd` code path at all.  I think we can look into refactoring the code when we have more than two modes.